client/my-sites/plans/ecommerce-trial/index.tsx

```diff
@@ -0,0 +1,389 @@
+							? translate(
+									'Your free trial has expired. Sign up to a plan to unlock new features and keep your store running.'
+							  )
+							: translate(
+									'Your free trial will end in %(daysLeft)d day. Sign up to a plan by %(expirationdate)s to unlock new features and keep your store running.',
+									'Your free trial will end in %(daysLeft)d days. Sign up to a plan by %(expirationdate)s to unlock new features and keep your store running.',
+									{
+										count: eCommerceTrialDaysLeftToDisplay,
+										args: {
+											daysLeft: eCommerceTrialDaysLeftToDisplay,
+											expirationdate: readableExpirationDate,
+										},
+									}
+							  ) }
+					</p>
+				</div>
+				<div className="e-commerce-trial-plans__chart-wrapper">
+					<DoughnutChart
+						progress={ trialProgress }
+						text={ eCommerceTrialDaysLeftToDisplay?.toString() }
+					/>
+					<br />
+					<span className="e-commerce-trial-plans__chart-label">
+						{ isTrialExpired
+							? translate( 'Your free trial has expired' )
+							: translate( 'day left in trial', 'days left in trial', {
+									count: eCommerceTrialDaysLeftToDisplay,
+							  } ) }
+					</span>
+				</div>
+			</Card>
+
+			<div className="e-commerce-trial-plans__interval-toggle-wrapper">
+				<SegmentedControl compact primary={ true } className={ segmentClasses }>
+					<SegmentedControl.Item
+						selected={ interval === 'monthly' }
+						onClick={ () => setInterval( 'monthly' ) }
+					>
+						<span>{ translate( 'Pay Monthly' ) }</span>
+					</SegmentedControl.Item>
+
+					<SegmentedControl.Item
+						selected={ interval === 'yearly' }
+						onClick={ () => setInterval( 'yearly' ) }
+					>
+						<span>{ translate( 'Pay Annually' ) }</span>
+					</SegmentedControl.Item>
+				</SegmentedControl>
+			</div>
+
+			<Card className="e-commerce-trial-plans__price-card">
+				<div className="e-commerce-trial-plans__price-card-text">
+					<span className="e-commerce-trial-plans__price-card-title">
+						{
+							// TODO: translate when final copy is available
+							'Commerce'
+						}
+					</span>
+					<span className="e-commerce-trial-plans__price-card-subtitle">
+						{ translate( 'Accelerate your growth with advanced features.' ) }
+					</span>
+				</div>
+				<div className="e-commerce-trial-plans__price-card-conditions">
+					<span className="e-commerce-trial-plans__price-card-value">
+						<span className="e-commerce-trial-plans__price-card-value-symbol">$</span>
+						{
+							// TODO: make it dynamic
+							'45'
+						}
+					</span>
+					<span className="e-commerce-trial-plans__price-card-interval">
+						{
+							// TODO: translate when final copy is available
+							'per month, $540 billed annually'
+						}
+					</span>
+					<span className="e-commerce-trial-plans__price-card-savings">
+						{
+							// TODO: translate when final copy is available
+							'SAVE 31% BY PAYING ANNUALLY'
+						}
+					</span>
```

Comment:
I think we should implement this now, but we should use at least some interpolation. Something like the following:
```suggestion
					const priceContent = isAnnualSubscription
						translate(
							'{{priceWrapper}}%(monthlyPrice)s{{/priceWrapper}} per month, %(annualPrice)s billed annually',
							{
								args: {
									monthlyPrice: '$45', // we should be able to compute both prices, including the symbol from the currentPlan object and/or a related selector
									annualPrice: '$540',
								},
								components: {
									priceWrapper: <span className="e-commerce-trial-plans__price-card-value" />,
								},
							}
						)
						: translate(
							'{{priceWrapper}}%(monthlyPrice)s{{/priceWrapper}} per month',
							{
								components: {
									priceWrapper: <span className="e-commerce-trial-plans__price-card-value" />,
								}
							}
						);
					const annualUpsellContent = ! isAnnualSubscription
						? null
						: (
							<span className="e-commerce-trial-plans__price-card-savings">
								{ translate(
									// apply upper-casing via CSS
									'Save %(percentageSavings)s%% by paying annually',
									{
										percentageSavings: 31, // compute this here
									}
								) }
							</span>
						);
					// ...
					{ priceContent }
					{ annualUpsellContent }
```